src/Resolution.cpp

```diff
@@ -40,15 +40,15 @@ static NSSize max_window_size(Gosu::Window* window)
         style = NSWindowStyleMaskBorderless;
     }
     else {
-        style = (NSWindowStyleMaskTitled|NSWindowStyleMaskClosable|NSWindowStyleMaskMiniaturizable);
+        style = (NSWindowStyleMaskTitled | NSWindowStyleMaskClosable | NSWindowStyleMaskMiniaturizable);
     }
     if (window && window->resizable()) {
         style |= NSWindowStyleMaskResizable;
     }
 
     auto index = window ? SDL_GetWindowDisplayIndex(Gosu::shared_window()) : 0;
     auto screen_frame = NSScreen.screens[index].visibleFrame;
-    return [NSWindow contentRectForFrameRect:screen_frame styleMask:style].size;
+    return[NSWindow contentRectForFrameRect : screen_frame styleMask : style].size;
```

Comment:
```suggestion
    return [NSWindow contentRectForFrameRect:screen_frame styleMask:style].size;
```
That's the normal way to format this in Objective-C